.github/workflows/cypress-participant-tests.yml

```diff
@@ -1,7 +1,13 @@
 name: Run Cypress Participant Tests
+
 on:
   pull_request:
     types: [opened, reopened, synchronize]
+  workflow_dispatch:
+    inputs:
+      pr_branch:
+        description: 'Branch to checkout and run tests against originating from the PR'
```

Comment:
Is this supposed to be origin instead of originating?

Reply:
Nope, just confusing grammar.

I'll change it to "Checkout this branch from the PR and execute CI/CD tests on it."